hotel-management/vite.config.ts

```diff
@@ -4,4 +4,9 @@ import react from '@vitejs/plugin-react';
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  resolve: {
+    alias: {
+      '@': '/src'
+    }
+  }
```

Comment:
I thought we check about configuration about alias in previous PR?

Reply:
If this was fixed the alias path, please create another PR about this fix. And feel free to merge this

Reply:
yes, I should've splitted the pr for this change, I will be careful next time.